internal/bundle/bundle.go

```diff
@@ -67,7 +68,7 @@ func Validate(ctx context.Context, imagePath string) (*Report, error) {
 			return nil, fmt.Errorf("%v", err)
 		}
 		if k8sVer, found := ocpToKubeVersion[targetVersion]; found {
-			log.L().Debugf("OpenShift %s detected in annotations. Running with additional checks enabled.", targetVersion)
+			logger.V(log.DBG).Info("running with additional checks enabled because of the OpenShift version detected", "version", targetVersion)
```

Comment:
It seems we no longer have a formatter, is this syntax doing a literal replacement of the word `version` in the sentence with the value `targetVersions`?

Reply:
This is a variadic method accepting an arbitrary number of values at the end, and it associates each odd parameter as a key with the following even parameters as a value (excluding the very first parameter). See: https://github.com/go-logr/logr/blob/master/logr.go#L270

Reply:
@komish I looked at this more after my question, and I see things how you explained them. I just don't love this syntax, but internally `logr` ends up internally converting to `fmt.Sprintf("Openshift %s, version)` anyways, and that's all that matters I guess.